scripts/tuple_tree_generator/tuple_tree_generator/templates/tuple_tree_gen.py.tpl

```diff
@@ -0,0 +1,108 @@
+###- This template file is distributed under the MIT License. See LICENSE.md for details. -###
+###- The notice below applies to the generated files. -###
+
+# This file is distributed under the MIT License. See LICENSE.md for details.
+# This file is autogenerated! Do not edit it directly
+import random
+import sys
+from dataclasses import dataclass, field
+from typing import List
+
+from .external import EnumBase, Reference, StructBase, no_default
+from .external import YamlLoader as _ExternalYamlLoader
+from .external import YamlDumper as _ExternalYamlDumper
+##- for t in generator.external_types ##
+from .external import 't'
+## endfor ##
+
+
+class YamlLoader(_ExternalYamlLoader):
+    pass
+
+
+class YamlDumper(_ExternalYamlDumper):
+    pass
```

Comment:
Did you understand if the registry is reset every time you inherit or what?

Reply:
Yes, I don't know exactly how, but running this
```python
import yaml

class YamlDumper1(yaml.Dumper):
    pass

class YamlDumper2(yaml.Dumper):
    pass

class MyClass:
    @classmethod
    def representer(cls, dumper: yaml.dumper.Dumper, instance):
        return dumper.represent_str("asdasd")


YamlDumper1.add_representer(MyClass, MyClass.representer)

instance = MyClass()

print(yaml.dump(instance, Dumper=YamlDumper1))
print(yaml.dump(instance, Dumper=YamlDumper2))
```
prints
```
asdasd
...

!!python/object:__main__.MyClass {}
```
because `YamlDumper2` does not have an associated representer for `MyClass`

Reply:
Gotcha